tlair/src/test/scala/at/forsyte/apalache/tla/lir/io/TestCounterexampleWriter.scala

```diff
@@ -10,30 +10,32 @@ import org.scalatest.FunSuite
 @RunWith(classOf[JUnitRunner])
 class TestCounterexampleWriter extends FunSuite {
 
-
-  def compare(kind: String, rootModule: TlaModule, notInvariant: NotInvariant, states: List[NextState], expected: String): Unit = {
+  def compare(kind: String, rootModule: TlaModule, notInvariant: NotInvariant, states: List[NextState],
+      expected: String): Unit = {
     val stringWriter = new StringWriter()
     val printWriter = new PrintWriter(stringWriter)
     val writer = CounterexampleWriter(kind, printWriter)
     writer.write(rootModule, notInvariant, states)
     printWriter.flush()
-    assert(stringWriter.toString.replaceFirst("Created by Apalache on .*\n", "Created by Apalache on DATETIME\n") == expected)
+    assert(stringWriter.toString.replaceFirst("Created by Apalache on .*\n",
+            "Created by Apalache on DATETIME\n") == expected)
   }
 
   test("single state") {
-    compare("tla",
-      new TlaModule("test",List()),
-      gt(name("x"), int(1)),
-      List(
-        ("",Map("x" -> int(2)))
-      ),
-      """------------------------- MODULE counterexample -------------------------
+    compare(
+        "tla",
+        new TlaModule("test", List()),
+        gt(name("x"), int(1)),
+        List(
+            ("", Map("x" -> int(2)))
+        ),
+        """------------------------- MODULE counterexample -------------------------
         |
         |EXTENDS test
         |
-        |(* Initial state *)
+        |(* Constant initialization state *)
         |
-        |State1 ==
+        |StateConst ==
```

Comment:
It's a strange test, as `x` is not a constant here but a variable.

Reply:
That's because your test is written in a way that doesn't reflect real-case use. The 1st element of `states`, in the test, is the `x=1` state, wehreas in the real tool case it would be `TRUE` or whatever `CInit` does.